Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Build hw-event-proxy binaries
+# FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
+FROM docker.io/openshift/origin-release:golang-1.15 AS go-builder
+ENV GO111MODULE=off
+ENV CGO_ENABLED=1
+ENV COMMON_GO_ARGS=-race
+ENV GOOS=linux
+ENV GOPATH=/go
+
+COPY ./scripts /scripts
+WORKDIR /go/src/github.com/redhat-cne/hw-event-proxy/hw-event-proxy
+COPY ./hw-event-proxy .
+RUN /scripts/build-go.sh
+
+# Build message-parser and install virtual environment
+FROM docker.io/centos:centos8 as python-builder
```

Comment:
may be can use ubi8 image with python 3.8 installed instead of centos?
https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a?container-tabs=overview

Reply:
Is ubi8 a preferred base? I tried it and found it is much easier to install dependencies but the final size is 936M vs 420M for centos8 based image.

Reply:
I see that is interesting I would have always thought UBI base images would result in the smallest footprint and would be relevant for the base image on L27
Although as this is a builder Image I was suggesting you could use an image that has python preinstalled instead of using base image and installing python in the dockerFile.. thoughts?

Reply:
Updated to use ubi8/python-38 as base. Image size is 936M but it is much simpler to build.